Sources/Plasma/Apps/plPythonPack/PythonInterface.cpp

```diff
@@ -79,7 +79,7 @@ void PythonInterface::initPython(const plFileName& rootDir, const std::vector<pl
         config.optimization_level = 2;
         config.write_bytecode = 0;
         config.user_site_directory = 0;
-        config.program_name = L"plasma";
+        PyConfig_SetString(&config, &config.program_name, L"plasma");
```

Comment:
Related: I learned later that PyConfig can leak memory and it needs to be freed with `PyConfig_Clear` 😢

Reply:
Ah, I was wondering how this didn't crash, since Python expects `program_name` to be dynamically allocated...  Not calling `PyConfig_Clear` would certainly explain that :)